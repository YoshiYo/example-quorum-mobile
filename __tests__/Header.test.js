import React from 'react';
import Header from '../src/components/header/header';
import { Text, View } from 'react-native'
import renderer from 'react-test-renderer';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });

import toJson from 'enzyme-to-json';
//import {createSerializer} from 'enzyme-to-json';
 
//expect.addSnapshotSerializer(createSerializer({mode: 'deep'}));


describe('Hender rendered', () => {
  it('Render the header and watch changes', () => {
    const wrapper = shallow(
      <Header />
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
  it('Render a component View', ()=> {
    const wrapper = shallow(
        <Header />
    );
    expect(wrapper.find(View));
  });
});







