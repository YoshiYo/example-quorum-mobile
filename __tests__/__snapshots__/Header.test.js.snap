// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hender rendered Render the header and watch changes 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#5353DE",
      "flex": 1,
      "height": 100,
      "justifyContent": "center",
      "padding": 40,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#FFF",
        "fontSize": 20,
      }
    }
  >
     Welcome on Quorum 
  </Text>
</Component>
`;
